tests-behat/bootstrap/Context.php

```diff
@@ -318,6 +305,40 @@ public function modalIsShowingText($arg1, $arg2)
         }
     }
 
+    /**
+     * Get a node element by it's selector.
+     * Will try to get element for 20ms.
+     * Exemple: Use with a modal window where reloaded content
+     * will resize it's window thus making it not accessible at first.
+     */
+    private function waitForNodeElement(string $selector): ?NodeElement
```

Comment:
I do not see any need for this method, when we need to wait, we wait with jquery wait which wait some extra time.

Reply:
same as above.

Reply:
same as above.